agent/src/main/java/com/teamscale/jacoco/agent/upload/azure/AzureFileStorageUploader.java

```diff
@@ -50,17 +59,34 @@ public AzureFileStorageUploader(AzureFileStorageConfig config, List<Path> additi
 		validateUploadUrl();
 	}
 
-	/** Extracts and returns the account of the provided azure file storage from the URL. */
+	@Override
+	public void markFileForUploadRetry(CoverageFile coverageFile) {
+		File uploadMetadataFile = new File(FileSystemUtils.replaceFilePathFilenameWith(
+				com.teamscale.client.FileSystemUtils.normalizeSeparators(coverageFile.toString()),
+				coverageFile.getName() + RETRY_UPLOAD_FILE_SUFFIX));
+		Properties properties = new Properties();
+		properties.setProperty(URL.toString(), this.uploadUrl.toString());
```

Comment:
No this makes sense, I will then just create an empty file for the azure uploader so that we know there is something, but we don't need to change configurations.